### PR TITLE
Fix Supabase SSR example

### PR DESCRIPTION
**Pull request by bmorrisondev**
> [!IMPORTANT]
> 🔎 Previews:
>
> - https://clerk.com/docs/pr/1489/integrations/databases/supabase#integrate-supabase-with-clerk

### Explanation:

The SSR version of the example code works in development but does not build. 

### This PR:

- This change updates the sample code with a version that does both.

* The sample repository has already been updated: https://github.com/clerk/clerk-supabase-nextjs/tree/main/src/app/ssr

### CODE REVIEW

**Comment by bmorrisondev** [comment]
@clerk/docs Linting is failing due to an issue with a doc unrelated to this PR. I'm open to fixing it, but wanted to check if that's OK with y'all first.

File: docs/integrations/databases/supabase.mdx [hunk 1]
```diff
@@ -366,10 +366,9 @@ For interacting with the Supabase dashboard, you can either use the **Supabase i
       The client can be used to query data on page load.
 
       ```tsx {{ filename: 'src/app/ssr/page.tsx' }}
-      import { Input } from '@/components/ui/input'
-      import { Button } from '@/components/ui/button'
-      import { createClerkSupabaseClient } from './client'
+      import { createClerkSupabaseClientSsr } from './client'
       import TaskRow from './TaskRow'
+      import AddTaskForm from './AddTaskForm'
```
[the remark this answers is not in the document]
Comment (bmorrisondev) [reply docs/integrations/databases/supabase.mdx:371]: I dont show `TaskRow` either, which is why I didnt add `AddTaskForm`.

Should both be added?

[the remark this answers is not in the document]
Comment (alexisintech) [reply docs/integrations/databases/supabase.mdx:371]: we want fully working, copy and pastable examples in the guide!

**Comment by alexisintech** [comment]
> @clerk/docs Linting is failing due to an issue with a doc unrelated to this PR. I'm open to fixing it, but wanted to check if that's OK with y'all first.

I'm working on it in a separate PR, don't worry about it

File: docs/integrations/databases/supabase.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (bmorrisondev) [reply docs/integrations/databases/supabase.mdx]: @alexisintech take another look and let me know what you think about it now.

**Review by alexisintech — APPROVED** [review]

**Pull request merged**